aiida/parsers/parser.py

```diff
@@ -15,16 +15,22 @@
 from __future__ import print_function
 from __future__ import absolute_import
 
+from abc import ABCMeta, abstractmethod
+import six
+
 from aiida.common import exceptions, extendeddicts
 from aiida.engine import calcfunction
 from aiida.engine.processes.ports import CalcJobOutputPort
 
 __all__ = ('Parser',)
 
 
-class Parser(object):  # pylint: disable=useless-object-inheritance
+@six.add_metaclass(ABCMeta)
+class Parser(object):
     """Base class for a Parser that can parse the outputs produced by a CalcJob process."""
 
+    # pylint: disable=useless-object-inheritance
```

Comment:
interesting that this works, putting it inside the class.
is there a reason why you moved it there?
it would seem the original place is closer to the problem it complains about

Reply:
Only reason is that all of a sudden the linter started breaking my balls and complaining about this. Moving it seemed to have done the job. Don't understand why